### DISCUSSION TITLE
how to change bond type in sdf file

### DISCUSSION THREAD

**Discussion by orgw** [rdkit/rdkit#5253] [Q&A]
![image](https://user-images.githubusercontent.com/64889801/166203993-ec7e0bee-dcab-425f-972c-87d686865eac.png)

Hi I'm a noob. The sdf file above is that consisting of a quinazoline

it seems that the bond type is recorded as double bond, whereas I want to change it to be "aromatic" bond. How would I change that in an sdf file??

Thanks

**Comment by greglandrum** [comment]
@orgw : According to the SDF file specification, aromatic bonds are only supposed to be used when defining queries.
Why do you want to make these changes to the file?
